meetingMinutes/notat_3_okt.txt

```diff
@@ -0,0 +1,42 @@
+Angående postgres database i bunnen:
```

Comment:
Notatene mine fra Rurik møtet. Hvis de ikke gir mening for deg så hiver jeg de ut.